src/util/suggestions-util.ts

```diff
@@ -4,35 +4,58 @@ type Config = {
   numOptions?: number;
   showCurrentWordAsLastSuggestion?: boolean;
   lang?: Language;
+  punctuationsToBeHandledAtEndOfWord?: string[];
 };
 
 export const getTransliterateSuggestions = async (
   word: string,
   config?: Config,
 ): Promise<string[]> => {
-  const { numOptions, showCurrentWordAsLastSuggestion, lang } = config || {
+  const {
+    numOptions,
+    showCurrentWordAsLastSuggestion,
+    lang,
+    punctuationsToBeHandledAtEndOfWord,
+  } = config || {
     numOptions: 5,
     showCurrentWordAsLastSuggestion: true,
     lang: "hi",
+    punctuationsToBeHandledAtEndOfWord: [",", '"'],
   };
   // fetch suggestion from api
   // const url = `https://www.google.com/inputtools/request?ime=transliteration_en_${lang}&num=5&cp=0&cs=0&ie=utf-8&oe=utf-8&app=jsapi&text=${word}`;
 
+  let punctuation = word.slice(-1);
+  if (punctuationsToBeHandledAtEndOfWord?.includes(punctuation)) {
+    word = word.slice(0, -1);
```

Comment:
issue: This does not handle the case of multiple punctuations at the end. e.g. `some "word", here`. -> `",`